File: python/AreaOfCircle.py
class AreaOfCircle:
    PI = 3.14159

    def __init__(self) -> None:
        pass

    def area(self, rad):
        area = AreaOfCircle.PI * pow(rad, 2);
        print(area);

    def perimeter(self, rad):
        para = 2 * AreaOfCircle.PI * rad;
        print(para);

File: java/src/com/java/exercises/AreaOfCircle.java
package com.java.exercises;

public class AreaOfCircle {
	static double PI = Math.PI;
	public static void area(double rad) {		
		double area = PI * rad * rad;
		System.out.printf("Area of a circle is: %f", area);
	}
	
	public static void perimeter(double rad) {
		double peri = 2 * PI * rad;
		System.out.printf("Perimeter of a circle: %.2f", peri);
	}
}
